control-plane/connect-inject/webhook/consul_dataplane_sidecar.go

```diff
@@ -120,13 +120,21 @@ func (w *MeshWebhook) consulDataplaneSidecar(namespace corev1.Namespace, pod cor
 			},
 			{
 				Name:  "DP_CREDENTIAL_LOGIN_META",
-				Value: "pod=$(POD_NAMESPACE)/$(POD_NAME),pod-uid=$(POD_UID)",
+				Value: "pod=$(POD_NAMESPACE)/$(POD_NAME)",
+			},
+			{
+				Name:  "DP_CREDENTIAL_LOGIN_META",
+				Value: "pod-uid=$(POD_UID)",
```

Comment:
Will this not overwrite the value of the environment variable setting the meta value `pod=`? Do you see both pieces of meta on the token after this change

Reply:
removed it. do you know why we have used the same value for DP_CREDENTIAL_LOGIN_META and DP_CREDENTIAL_LOGIN_META1 ?

Reply:
Ah, so I'm seeing [here](https://github.com/hashicorp/consul-dataplane/blob/main/cmd/consul-dataplane/flags.go#L47-L56) that logic allows the environment variable to be specified with a suffix of {1-9} so that you can specify multiple values. So I think the key that makes this PR work is DP_CREDENTIAL_LOGIN_META2 where you've added the pod uid.